### PR TITLE
feat: added all modal component 

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added modals component for createpsot, editcommentmodalpost, postmodalpost, usercommentmodalpost and editProfilemodalpost

Files to check 👍 

src/components/modal/CreatePost.tsx
 src/components/modal/EditCommentModal.tsx
 src/components/modal/EditProfileModal.tsx
 src/components/modal/PostModal.tsx
 src/components/modal/UserCommentModal.tsx
 src/components/modal/styles/postmodal-style-props.ts
 src/components/modal/styles/usercommentmodal-style-props.ts
  src/components/modal/types/CreatePost.ts
 src/components/modal/types/EditProfileModalType.ts
 src/components/modal/types/PostModal.ts
 src/components/modal/types/index.ts

Live preview : https://demo-ten-cyan.vercel.app/

### CODE REVIEW

**Review by MaruthiPothuganti — APPROVED** [review]
Awesome UI and clean code. Modals work fine. PR looks good to merge.

File: src/components/modal/PostModal.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          <ModalHeader color={cardText}>Filter</ModalHeader>
+
+          <ModalCloseButton color={cardText} />
+          <Divider />
+
+          <ModalBody>
+            {/* <Text color="pink.500" {...textStyleProps}>
+              Unfollow
+            </Text> */}
+
```
Comment (anjali1102) [inline src/components/modal/PostModal.tsx:39]: you can remove these

**Review by anjali1102 — APPROVED** [review]

File: src/components/modal/PostModal.tsx (hunk 1 above)
Comment (mehulsatardekar) [reply src/components/modal/PostModal.tsx:39]: I might add this feature later on

**Review by prath-dev — APPROVED** [review]
all functionalities are working proper looks goods to merge

**Pull request merged**